### DISCUSSION TITLE
[Auth] password reminder

### DISCUSSION THREAD

**Discussion by sciamano89** [ente-io/ente#2731] [Enhancements]
A simple option to prompt us for our password every 14 days, but without locking us out of the account if we get it wrong. So this should be an _option_ in the settings AND when prompted we should have the option to ‘skip’. 

We can all agree that we’re here to improve our security, which means we’re most likely choosing a strong password _unique_ to Auth. 
Without the app prompting us to input the password on a regular basis, people might end up forgetting it (I know I have in the past…I have a general idea of what the password is, but not the specifics, like my punctuation choices for example).

**Comment by maroofmedia** [comment]
+1 for this idea! Along with the 14-day prompt, it would be great to add additional triggers for the master password request. For instance, prompting for the password upon device reboot or restart would further enhance security. This way, we can strike a balance between security and convenience, ensuring that users remember their passwords without feeling locked out.

**Comment by mrx23dot** [comment]
Or simply use a 2nd offline password manager like Keepass, or a piece of paper in vault, for rest of times use fingerprint.
